src/portal/portal.services.ts

```diff
@@ -36,180 +38,195 @@ export class PortalService {
           result: null,
         });
       } else {
-        const sb_rc_search = await this.sbrcService.sbrcSearchEL('TeacherV1', {
```

Comment:
@YoginiTayade  remove all commented code from this function

Reply:
Resolved